e2e/src/orbs-client.ts

```diff
@@ -57,25 +55,29 @@ export class OrbsClientSession {
   }
 
   async call(contractAddress: string, payload: string) {
-    const { resultJson } = await this.orbsClient.call({
+    const { resultJson } = await this.orbsClient.callContract({
       sender: this.getAddress(),
-      contractAddress: contractAddress,
+      contractAddress: {address: contractAddress},
       payload: payload
     });
     return JSON.parse(resultJson);
   }
 
   public generateTransaction(contractAddress: string, payload: string): Transaction {
     return {
-      version: 0,
-      sender: this.senderAddress,
-      contractAddress: contractAddress,
-      payload: payload,
-      signature: "" // TODO: add a signature once implement in the network-side
+      header: {
+        version: 0,
+        sender: this.getAddress(),
+        sequenceNumber: 0
```

Comment:
The `sequence` here is confusing, compared how it's used in other blockchains (e.g., Ethereum, Stellar, Ripple, etc.). Maybe something with `order` instead?

Reply:
Maybe. Let's wait for the final spec and change accordingly